### PR TITLE
Refactor READEME

### PR DESCRIPTION
**Pull request by zytx121**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily get feedback. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation

- [x] update README
- [x] synchronize commits in 0.x. #396 #428 

## Modification

Please briefly describe what modification is made in this PR.

## BC-breaking (Optional)

Does the modification introduce changes that break the back-compatibility of the downstream repos?
If so, please describe how it breaks the compatibility and how the downstream projects should modify their code to keep compatibility with this PR.

## Use cases (Optional)

If this PR introduces a new feature, it is better to list some use cases here, and update the documentation.

## Checklist

1. Pre-commit or other linting tools are used to fix the potential lint issues.
2. The modification is covered by complete unit tests. If not, please add more unit test to ensure the correctness.
3. The documentation has been modified accordingly, like docstring or example tutorials.

### CODE REVIEW

**Comment by yangxue0827** [comment]
citation
```

@inproceedings{zhou2022mmrotate,
  title   = {MMRotate: A Rotated Object Detection Benchmark using PyTorch},
  author  = {Zhou, Yue and Yang, Xue and Zhang, Gefan and Wang, Jiabao and Liu, Yanyi and
             Hou, Liping and Jiang, Xue and Liu, Xingzhao and Yan, Junchi and Lyu, Chengqi and
             Zhang, Wenwei and Chen, Kai},
  booktitle={Proceedings of the 30th ACM International Conference on Multimedia},
  pages = {7331–7334},
  numpages = {4},
  year={2022}

```

File: README.md [hunk 1]
```diff
@@ -72,41 +72,49 @@ https://user-images.githubusercontent.com/10410257/154433305-416d129b-60c8-44c7-
...
 
-**0.3.2** was released in 6/7/2022:
+**v1.0.0rc0** was released in 11/11/2022:
 
-- Support Oriented Reppoints (CVPR'22) (#286)
-- Support ConvNeXt backbone (CVPR'22) (#343)
-
-Please refer to [changelog.md](docs/en/changelog.md) for details and release history.
+- Unifies interfaces of all components based on [MMEngine](https://github.com/open-mmlab/mmengine).
+- Faster training and testing speed with complete support of mixed precision training.
```
Comment (RangiLyu) [inline README.md:78]: Did you benchmark?

File: docs/en/notes/faq.md [hunk 2]
```diff
@@ -11,8 +11,8 @@ Compatible MMCV, MMDetection and MMRotate versions are shown as below. Please in
 | MMRotate version |       MMCV version        | MMDetection version |
 | :--------------: | :-----------------------: | :-----------------: |
 |     dev-1.x      |    mmcv-full>=2.0.0rc1    |  mmdet >= 3.0.0rc0  |
```
Comment (RangiLyu) [inline docs/en/notes/faq.md:13]: mmdet has box type after 3.0.0rc1, can mmrotate use rc0?

File: mmrotate/__init__.py [hunk 3]
```diff
@@ -17,7 +17,7 @@
     f'<= {mmcv_maximum_version} instead.'
 
 mmengine_minimum_version = '0.0.0'
```
Comment (RangiLyu) [inline mmrotate/__init__.py:19]: should update minimum version

**Review by RangiLyu — APPROVED** [review]

**Pull request merged**